lib/Local.js

```diff
@@ -199,10 +245,10 @@ function Local(){
         conf.proxyHost = this.proxyHost;
         conf.proxyPort = this.proxyPort;
       }
-      this.binary.binaryPath(conf, callback);
+      return callback ? this.binary.binaryPath(conf, callback) : this.binary.binaryPath(conf);
```

Comment:
return in case of callback doesn't make sense. Keep this in if else block and return only in case of synchronous call. Same for other similar instances in PR